### PR TITLE
Clarify handling of #rrggbb(aa) colors in matplotlibrc.

### PR DESCRIPTION
**Pull request by anntzer**
The inconsistency is not that great, but let's at least document it.

See https://discourse.matplotlib.org/t/specifying-hex-colors-in-style-sheets/22624.

## PR Checklist

**Tests and Styling**
- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).

**Documentation**
- [ ] New features are documented, with examples if plot related.
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

File: lib/matplotlib/rcsetup.py [hunk 1]
```diff
@@ -312,10 +312,12 @@ def validate_color(s):
     if isinstance(s, str):
         if s.lower() == 'none':
             return 'none'
-        if len(s) == 6 or len(s) == 8:
-            stmp = '#' + s
-            if is_color_like(stmp):
-                return stmp
+        # Support "rrggbb"/"rggbbaa" without leading '#' because these would be
```
Comment (pepe78) [inline lib/matplotlib/rcsetup.py:315]: rggbbaa -> rrggbbaa

it's just in comments, but...

Comment (anntzer) [reply lib/matplotlib/rcsetup.py:315]: thanks for noting that, fixed.

**Comment by anntzer** [comment]
No strong opinion there (other than my oft-stated opinion about the matplotlibrc syntax in general).

**Comment by timhoffm** [comment]
Superseeded by #22589.

**Pull request closed without merging**